src/compiler/c.rs

```diff
@@ -100,6 +100,8 @@ pub struct ParsedArguments {
     pub common_args: Vec<OsString>,
     /// Commandline arguments for the compiler that specify the architecture given
     pub arch_args: Vec<OsString>,
+    /// Commandline arguments for the preprocessor or the compiler that don't affect the computed hash.
+    pub unhashed_args: Vec<OsString>,
```

Comment:
I'd like to have another statement, that none of these arguments listed may in any circumstance change the resulting binary.